### PR TITLE
Add 2020-07-02 meeting notes (closes #184)

### LINKED ISSUE #184: TSC meeting 02-July-2020
# Time

UTC Thu 02-Jul-2020 20:00:
- Los Angeles: Thu 02-Jul-2020 13:00
- Chicago: Thu 02-Jul-2020 15:00
- New York: Thu 02-Jul-2020 16:00
- Madrid: Thu 02-Jul-2020 22:00
- Moscow: Thu 02-Jul-2020 23:00
- Tokyo: Fri 03-Jul-2020 05:00
- Sydney: Fri 03-Jul-2020 06:00

# Location

https://gitter.im/eslint/tsc-meetings

# Agenda

Extracted from:

* Issues and pull requests from the ESLint organization with the ["tsc agenda" label](https://github.com/issues?utf8=%E2%9C%93&q=org%3Aeslint+label%3A%22tsc+agenda%22)
* Comments on this issue

# Invited

- Nicholas C. Zakas (@nzakas) - TSC
- Brandon Mills (@btmills) - TSC
- Toru Nagashima (@mysticatea) - TSC
- Kai Cataldo (@kaicataldo) - TSC

# Public participation

Anyone is welcome to attend the meeting as observers. We ask that you refrain from interrupting the meeting once it begins and only participate if invited to do so.

### PR DESCRIPTION
**Pull request by btmills**
🚧 this does not yet include the transcript. @kaicataldo - let me know how your automation for that is coming along and whether I should run it manually and attach to this PR.

@kaicataldo I structured the notes for the config cloning discussion so that copying the blockquoted parts should give you a decent start on tomorrow's release announcement with links included assuming everyone's happy with the notes.

### CODE REVIEW

**Comment by kaicataldo** [comment]
@btmills Writing up the GitHub Action right now - would be cool if we could see if it works!

**Comment by kaicataldo** [comment]
@btmills PR is up here if you have some time to take a look! https://github.com/eslint/tsc-meetings/pull/186

File: notes/2020/2020-07-02.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+## Transcript
+
+[`2020-07-02-transcript.md`](2020-07-02-transcript.md)
+
+## Attending
+
+* Kai Cataldo (@kaicataldo) - TSC
+* Brandon Mills (@btmills) - TSC
+* Nicholas C. Zakas (@nzakas) -TSC
+* Toru Nagashima (@mysticatea) - TSC
```
Comment (kaicataldo) [inline notes/2020/2020-07-02.md:12]: Should we remove this line since @mysticatea was unable to make it?

Comment (btmills) [reply notes/2020/2020-07-02.md:12]: Indeed, good catch.

```suggestion
```

**Comment by kaicataldo** [comment]
Here we goooo

**Pull request merged**